settings.gradle

```diff
@@ -107,35 +107,30 @@ include 'serialization'
 gradleEnterprise {
     server = gradleEnterpriseUrl
     allowUntrustedServer = false
+    def apiKey = settings.ext.find('CORDA_GRADLE_SCAN_KEY') ?: System.getenv('CORDA_GRADLE_SCAN_KEY')
+    accessKey = apiKey
```

Comment:
is accessKey used at all ?

Reply:
@yarivus yes it's used internally by the `gradleEnterprise`  plugin, which provides this closure `accessKey` is a property which is provided by the plugin a user may set for authentication.

this is the same as other vars you see here such as `server ` / `allowUntrustedServer `

Reply:
It is used within Gradle. You can search for accessKey in Gradle documentation [here](https://docs.gradle.com/enterprise/gradle-plugin/#using_the_gradle_enterprise_connector).